website/docs/plugin/testing/acceptance-tests/index.mdx

```diff
@@ -236,37 +236,38 @@ jobs:
+| `TF_ACC_STATE_LINEAGE`       | N/A                                                                           | Set to `1` to enable state lineage debug logs, which are normally suppressed during acceptance testing.                                                                                                                                                                                                                                                                                                                                                              |
+| `TF_ACC_TEMP_DIR`            | Operating system specific via [`os.TempDir()`](https://pkg.go.dev/os#TempDir) | Set a temporary directory used for testing files and installing Terraform CLI, if installation is required.                                                                                                                                                                                                                                                                                                                                                          |
+| `TF_ACC_TERRAFORM_PATH`      | N/A                                                                           | Set the path to a Terraform CLI binary on the local filesystem to be used during testing. It must be executable. If not found and `TF_ACC_TERRAFORM_VERSION` is not set, an error is returned.                                                                                                                                                                                                                                                                       |
+| `TF_ACC_TERRAFORM_VERSION`   | N/A                                                                           | Set the exact version of Terraform CLI to automatically install into `TF_ACC_TEMP_DIR`. For example, `1.1.6` or `v1.0.11`.                                                                                                                                                                                                                                                                                                                                           |
+| `TF_ACC_PERSIST_WORKING_DIR` | N/A                                                                           | Set to any value to enable persisting the working directory and the files generated during execution of each `TestStep`. The location of each directory is output in the log for each `TestStep`.                                                                                                                                                                                                                                                                    |
```

Comment:
We could also consider setting up actual DEBUG/INFO logging entries as well within `copyWorkingDir` -- I do actually like the current `t.Logf()` usage though so folks don't _have_ to enable `TF_LOG` or friends.

```suggestion
| `TF_ACC_PERSIST_WORKING_DIR` | N/A                                                                           | Set to any value to enable persisting the working directory and the files generated during execution of each `TestStep`. The location of each directory is written to the test output for each `TestStep` when the `go test -v` (verbose) flag is provided.                                                                                                                                                                                                                                                                    |
```